svsim/src/main/scala/Workspace.scala

```diff
@@ -28,7 +30,7 @@ final class Workspace(
 ) {
 
   val absolutePath =
-    if (path.startsWith("/"))
+    if (Paths.get(path).isAbsolute())
```

Comment:
Nice. This is better than getting a separator and using that to check if absolute. 👍